pkg/tracing/attributes_test.go

```diff
@@ -43,3 +46,85 @@ func TestTriggerMessagingDestination(t *testing.T) {
 		t.Errorf("unexpected messaging destination: want %q, got %q", want, got)
 	}
 }
+
+func TestMessagingMessageIDAttribute(t *testing.T) {
+	type args struct {
+		ID string
+	}
+	tests := []struct {
+		name string
+		args args
+		want trace.Attribute
+	}{
```

Comment:
Can you expand args and use the ID directly?

Reply:
ok